cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -19,7 +22,7 @@ public class Mvn extends LocalToolCommandlet {
    */
   public Mvn(IdeContext context) {
 
-    super(context, "mvn", Set.of(TAG_JAVA, TAG_BUILD));
+    super(context, "mvn", Set.of(TAG_JAVA, TAG_BUILD, TAG_IDE));
```

Comment:
Maven is NOT and IDE (Integrated Development Environment). Therefore this tag does not make sense here.